### DISCUSSION TITLE
Admins panel on a subdomain

### DISCUSSION THREAD

**Discussion by WinglessWhistle** [YOURLS/YOURLS#3092] [Q&A] [answered]
Hello, Is it possible to have the main installation of YOURLS on a domain (EG: example.com) and have the admin panel on a subdomain (EG: panel.example.com), the domain I would use would be dedicated for YOURLS only.

**Comment by ozh** [comment, marked as answer]
It is certainly possible with some server configuration. Having a subdirectory matching a subdomain is a matter of server setup and/or rewrite rules, but nothing specific to YOURLS
